nimble/controller/include/controller/ble_ll_sched.h

```diff
@@ -31,13 +31,13 @@ extern "C" {
 /*
  * Worst case time needed for scheduled advertising item. This is the longest
  * possible time to receive a scan request and send a scan response (with the
- * appropriate IFS time between them). This number is calculated using the
+ * appropriate IFS time between them) on LE Uncoded PHY 1M. This number is calculated using the
```

Comment:
this is about legacy PDUs and those are on 1M by definition so not sure if this clear anything

Reply:
then we should say it is legacy adv obviously in this sentence.

Reply:
Yeap, I'm fine with emphasizing that this is for legacy PDUs.

Reply:
then what will be the next step?

Reply:
Please update PR with comment changes we discussed